src/wp-includes/rest-api.php

```diff
@@ -1284,12 +1285,36 @@ function rest_validate_value_from_schema( $value, $args, $param = '' ) {
 			return new WP_Error( 'rest_invalid_param', sprintf( __( '%1$s is not of type %2$s.' ), $param, 'object' ) );
 		}
 
+		/*
+		 * @ticket 48818
+		 */
+		if ( isset( $args['properties'] ) && ! empty( $value ) ) {
+			if ( isset( $args['required'] ) ) { // schema version 4
+				foreach ( $args['required'] as $name ) {
+					if ( ! isset( $value[ $name ] ) ) {
```

Comment:
```suggestion
					if ( ! array_key_exists( $name, $value ) ) {
```

`null` can be a valid value for a property so we need to use `array_key_exists`. This should also be adjusted in the v3 format.

Reply:
Done